pandas/tests/indexing/test_partial.py

```diff
@@ -397,14 +397,14 @@ def f():
         tm.assert_frame_equal(f(), expected)
 
         def f():
-            df = DataFrame()
+            df = empty_frame
```

Comment:
```suggestion
            df = empty_frame.copy()
```

since these mutate `empty_frame` probably want to `.copy()` (applies to next func as well)